packages/Toast/src/Toast.styles.js

```diff
@@ -0,0 +1,97 @@
+import styled, { css } from "styled-components";
+
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+import Button from "@paprika/button";
+import { visuallyHidden } from "@paprika/stylers/lib/includes";
+
+import Kinds from "./ToastKinds";
+
+const backgroundColors = {
+  [Kinds.SUCCESS]: tokens.color.greenLighten50,
+  [Kinds.WARNING]: tokens.color.yellowLighten30,
+  [Kinds.ERROR]: tokens.color.orangeLighten40,
+  [Kinds.INFO]: tokens.color.cremeLighten5,
+  [Kinds.LOCKED]: tokens.color.yellowLighten30,
+};
+
+// Calculated by brightness.
+const borderColors = {
+  [Kinds.SUCCESS]: "#c5e2d3",
```

Comment:
Can we use `greenLighten40` instead? The value is `#c4e3d3`... close enough for me!  I didn't check the others, but if there's a close enough token for each, that would be preferable.